Monika After Story/game/script-topics.rpy

```diff
@@ -4910,3 +4910,218 @@ label monika_asks_family:
+                    m "Don't be such a tease [player]~"
+                    m "Ahaha~"
+                "You don't have to force yourself.":
+                    m "I'm not really forcing myself [player]."
+                    m "I just need a bit of time to gather my thoughts."
+                    m "..."
+                "Okay, everyone! It's time to explain yourself~":
+                    m "Pfft!"
+                    m "Ahahahaha!"
+                    m "That's a nice impression of me [player]!"
+                    m "So, where was I?{w} Oh yeah! {i}explaining myself{/i}~"
+                    m "Ahaha~"
+            m "You know that I love it when you come to spend time with me right? And I'm always happy waiting for you to come here."
+            m "But it get's boring sometimes and I pass the time. You get that, right [player]?
+            m "This is kinda awkward.{w}Umm...{w}Movies! Where was I?"
+        "You mean like adaptations of your favourite books?":
+            m "Well, usually yes."
+            m "But I also watch regular movies from time to time."
+            m "There are some pretty interesting movies out there too."
+            m "For example."
+    m "While you were gone, I watched this movie called '{i}Whiplash{/i}'."
+    m "It was pretty good."
+    m "Maybe you know the movie too [player]?"
+    menu:
+        "Yes."
+            m "Oh, really now?"
+            m "Would you mind if I test you then?"
+            menu:
+                "Bring it!":
+#I need a way to bring the topic to when she asks later "want a synopsis?" if you get a question wrong. The questions are pretty basic, just what's the name of the main caracters.#
+                    m "Alright! Let's start!"
+                    m "What was the name of the MC?"
+                    menu:
+                        "Andrew":
+                            m "Correct!"
+                            m "Or maybe it was just a lucky guess? Okay, here's another one [player!"
+                            m "What's the name of the main antagonist?"
+                            menu:
+                                "Tanner":
+                                    m "Wrong! Caught you [player]!"
+                                    m "Ahaha!"
+#This is where I need the code or a way to somehow jump to another part of this topic#
+                                "Fletcher":
+                                    m "That's Right!"
+                                    m "I guess you did watch it."
+                                    m "Sorry for doubting you [player]."
+                                    menu:
+                                        "It's fine.":
+                                            m "Sorry I ever doubted you [player]."
+                                            m "So, what'd you think of the movie?"
+                                            menu:
+                                                "Pretty good."
```

Comment:
missing `:` at the end of `"Pretty good."`

Reply:
I'm fixing this as i go along so I hope you don't mind all the dones i'm sending. Done